### ISSUE TITLE
[Summary] Ability to pass in initial state

### ISSUE THREAD

**Issue by CompRhys** [state-spaces/mamba#175]
Several issues have requested the ability to input initial state but several of these have often been closed by those posting without the issue being resolved. This issue simply collates those prior issues to make comments by the maintainers in response to those more findable when searching open issues. 

https://github.com/state-spaces/mamba/issues/155
https://github.com/state-spaces/mamba/issues/146
https://github.com/state-spaces/mamba/issues/141
https://github.com/state-spaces/mamba/issues/127
https://github.com/state-spaces/mamba/issues/101

tldr; this functionality is work in progress

**Comment by CompRhys** [comment]
https://github.com/state-spaces/mamba/issues/258

**Comment by radarFudan** [comment]
Actually, if you don't mind 10x slower and 2x gpu memory usage, there is a workaround for now: https://github.com/state-spaces/mamba/issues/51

But I guess the true mamba with initial hidden states will require CUDA master to improve it.
